psocake/ImagePanel.py

```diff
@@ -305,8 +305,8 @@ def updatePolarizationFactor(self): # FIXME: change to vertical polarization aft
 
     def updateDetectorCentre(self):
         self.parent.cy, self.parent.cx = self.parent.det.point_indexes(self.parent.evt, pxy_um=(0, 0), pix_scale_size_um=None,
-                                                       xy0_off_pix=None,
-                                                       cframe=gu.CFRAME_PSANA, fract=True)
+                                                       xy0_off_pix=None, fract=True)
```

Comment:
should this be try/except consistent with other changes in this PR?

Reply:
yes I realized after the fact, it should be fine now